cmd/release-controller/http.go

```diff
@@ -52,6 +52,7 @@ const htmlPageEnd = `
 
 const releasePageHtml = `
 <h1>Release Status</h1>
+<div class="alert alert-warning">This site is part of OpenShift's continuous delivery pipeline. Neither the builds linked here nor the upgrade paths tested here are officially supported. Please visit <a href="https://access.redhat.com/downloads/content/290/">the Red Hat Customer Portal</a> for the latest supported product details.</div>
```

Comment:
This sentence should be part of the second paragraph below "... Visualize upgrades: ".  I do not think you should put the link to the portal because this is also used for OKD, and we should never send OKD users to the portal.

Reply:
Yeah, that's fine with me. We don't need to tell them where to go, just that this isn't the place.

Reply:
I have removed the link.